qark/plugins/logging_plugin.py

```diff
@@ -0,0 +1,117 @@
+import sys, os, re
+
+sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
+
+from yapsy.IPlugin import IPlugin
+from plugins import PluginUtil
+from modules import common
+from lib.pubsub import pub
+import lib.plyj.parser as plyj
+import lib.plyj.model as m
+
+class LoggingIssuesPlugin(IPlugin):
+
+    debug_regex = r'Log\.d'
+    verbose_regex = r'Log\.v'
+
+    def target(self, queue):
+        # get all decompiled files that contains usage of WebView
+        files = common.java_files
+        global parser, tree, fileName, verbose, debug, string_filename_d, string_filename_v
+        parser = plyj.Parser()
+        len_d = []
```

Comment:
As discussed, consider renaming the variables something more descriptive which makes it obvious to the reader what these refer to.